packages/velo-external-db-core/lib/service/schema_aware_data.js

```diff
@@ -85,6 +87,11 @@ class SchemaAwareDataService {
                     .map(i => unpackDates(i))
     }
 
+    async createProjectionArr(collectionName, projection) {
```

Comment:
function name can be better, i suggest schemaFieldNamesFor

Reply:
I think my function name isn't good as well, but your suggestion is good for now because it's internal, but when we'll get the projection from outside it will not choose all the schema field names for the collection. 
the purpose of the function is to clean up the projection it receives from non-existent fields, and if the projection is not valid - return the default (all the schema fields).

Reply:
Do you want me to treat it as internal only  for now, and remove this logic?

Reply:
in our case, if the fields would be invalid we will reject the request and not try to fix it.